app/index.html

```diff
@@ -33,29 +39,22 @@
               <span class="icon-bar"></span>
             </button>
 
-            <a class="navbar-brand" href="#/">todo</a>
+            <a class="navbar-brand" href="#/">Todo List</a>
           </div>
 
           <div class="collapse navbar-collapse" id="js-navbar-collapse">
 
             <ul class="nav navbar-nav">
-              <li class="active"><a href="#/">Home</a></li>
-              <li><a ng-href="#/about">About</a></li>
-              <li><a ng-href="#/">Contact</a></li>
+              <li><a href="#/">Current List</a></li>
+              <li><a ng-href="#/auth">Auth</a></li>
```

Comment:
Either ng-href or href will work for this. Choose one and have both 48 and 49 the same.